cmd/options.go

```diff
@@ -26,14 +26,18 @@ import (
 	"github.com/gardener/etcd-backup-restore/pkg/snapstore"
 
 	"github.com/gardener/etcd-backup-restore/pkg/initializer/validator"
-	"github.com/gardener/etcd-backup-restore/pkg/snapshot/restorer"
 
 	"github.com/gardener/etcd-backup-restore/pkg/server"
+	brtypes "github.com/gardener/etcd-backup-restore/pkg/types"
 	"github.com/ghodss/yaml"
 	"github.com/sirupsen/logrus"
 	flag "github.com/spf13/pflag"
 )
 
+const (
+	defaultCompactContainer = "compctedsnap.bkp"
```

Comment:
Is this needed if the default behaviour is to save the compacted snapshot in the current working directory?

Reply:
Not needed.. Missed it. Updated the PR with the necessary changes